### ISSUE TITLE
nuxt app instance unavailable error on all pages

### ISSUE THREAD

**Issue by antlionguard** [nuxt/framework#3904]
Sorry for wrongly opened issue. I cannot edit labels. This issue related with ```bridge``` 🙏 

> @antlionguard Thanks - following up with a PR on that mixin: #3900.

There's something broke after this PR. I got this two error in my all pages. @danielroe 

Firstly i got this:
<img width="769" alt="resim" src="https://user-images.githubusercontent.com/26603638/160143205-b954ecda-25ec-4a1c-8bf5-b822fc13753b.png">

and after refresh i got this:
<img width="780" alt="resim" src="https://user-images.githubusercontent.com/26603638/160143321-b7c2668f-5fc6-4883-8d30-948335c6a357.png">

_Originally posted by @antlionguard in https://github.com/nuxt/framework/issues/3431#issuecomment-1079097686_

**Comment by antlionguard** [comment]
This error still exist after #3906. Am i doing something wrong @danielroe? Sorry for a lot of interruptions 🙏 
 
<img width="753" alt="resim" src="https://user-images.githubusercontent.com/26603638/160159122-e2b7d75e-fa46-41e1-81fe-741fb4e714fa.png">

**Comment by danielroe** [comment]
It looks like I need a reproduction :pray

**Comment by antlionguard** [comment]
> It looks like I need a reproduction 🙏

i cannot reproduce but i found where the problem is came from. 

i closed the pinia from build modules
<img width="353" alt="resim" src="https://user-images.githubusercontent.com/26603638/160173352-dfefa755-6787-425e-ad06-80225576363f.png">

and there's no error here. i think @posva can help us? @danielroe

**Comment by Luferov** [comment]
Same.

**Comment by Luferov** [comment]
> It looks like I need a reproduction pray

https://github.com/devind-team/eleden

cd client
yarn
yarn run dev

**Comment by Luferov** [comment]
Completely impossible to work because of this error :(

**Comment by antlionguard** [comment]
> Completely impossible to work because of this error :(

yes same for me. i installed old version. @danielroe any update for this error?

**Comment by Luferov** [comment]
I created a simple project with nuxt-bridge & nuxt/pinia. https://github.com/Luferov/bea. Maybe the problem is not in nuxt-bridge, but in pinia/nuxt? @danielroe is it possible?
Although it worked before.

**Comment by antlionguard** [comment]
> I created a simple project with nuxt-bridge & nuxt/pinia. https://github.com/Luferov/bea. Maybe the problem is not in nuxt-bridge, but in pinia/nuxt? @danielroe is it possible? Although it worked before.

i think it's possible. but this error occurs after #3900. and #3900 resolves #3431.

**Comment by danielroe** [comment]
This will likely be resolved with https://github.com/vuejs/pinia/pull/791.

For now I think you can work around this with:

```js
import { defineNuxtConfig } from '@nuxt/bridge'

export default defineNuxtConfig({
  modules: ['@pinia/nuxt'],
  build: {
    transpile: ['pinia'],
  },
})
```

Let me know if that doesn't work.

**Comment by baixiaoyu2997** [comment]
@danielroe  Thanks , it's worked

**Comment by antlionguard** [comment]
> This will likely be resolved with [vuejs/pinia#791](https://github.com/vuejs/pinia/pull/791).
> 
> For now I think you can work around this with:
> 
> ```js
> import { defineNuxtConfig } from '@nuxt/bridge'
> 
> export default defineNuxtConfig({
>   modules: ['@pinia/nuxt'],
>   build: {
>     transpile: ['pinia'],
>   },
> })
> ```
> 
> Let me know if that doesn't work.

that's not worked. :(

But somethings it weird now. I tried on workarounding a any page. I didn't get this error if i remove all nuxt things(like useLazyAsyncData, auto imported composables and pinia stores). If i use any one of these i got error again.

<img width="641" alt="image" src="https://user-images.githubusercontent.com/26603638/161212398-05b25df8-aceb-4321-9397-d18c3cf0e5be.png">

**Comment by Luferov** [comment]
Unfortunately, the same effect :( I do commit to reproduction https://github.com/Luferov/bea.

**Comment by Luferov** [comment]
And got this warning, @danielroe should I create a new issue or not about this or not?
![image](https://user-images.githubusercontent.com/26242554/161244273-7a7c4a0f-3ee8-4491-82e2-b33fdd4b1076.png)

**Comment by danielroe** [comment]
@Luferov Yes, it's a different issue. However I think that you just need to ensure you're on the latest version of `unimport`. (If that doesn't resolve it, then do raise a new issue.)

**Comment by NidMo** [comment]
> @Luferov Yes, it's a different issue. However I think that you just need to ensure you're on the latest version of `unimport`. (If that doesn't resolve it, then do raise a new issue.)

I wonder if you have solved this problem, because I seem to have the same problem

![image](https://user-images.githubusercontent.com/24955673/161372032-2619dc8d-58c8-4e46-be1b-c1cb7b23eb91.png)
